src/app/lib/utilities/preprocessor/rules/timestamp.test.js

```diff
@@ -55,7 +78,106 @@ const headlineBlock = {
   },
 };
 
+const aresMediaBlock = {
```

Comment:
Also not sure what an aresMediaBlock should look like.

Reply:
There is an example of an `aresMediaBlock` here in the Video fixture data https://github.com/bbc/simorgh/blob/latest/src/app/containers/Video/fixtureData.jsx#L7